compiler/rustc_expand/src/mbe/macro_rules.rs

```diff
@@ -156,6 +156,7 @@ impl<'a> ParserAnyMacro<'a> {
 }
 
 struct MacroRulesMacroExpander {
+    id: NodeId,
```

Comment:
```suggestion
    node_id: NodeId,
```
Could you also replace the previously existing `is_local` field with this, instead of adding a new field?